test/TerminalLogger.jl

```diff
@@ -248,13 +250,51 @@ import TerminalLoggers.default_metafmt
     ⊏(s, re) = match(re, s) !== nothing
 
     @test genmsg("", progress=0.1, width=60) ⊏
-    r"Progress:  10%\|██.                  \|  ETA: 0:00:[0-9][0-9]"
+    r"Progress:  10%\|█+.* +\|  ETA: .*"
     @test genmsg("", progress=NaN, width=60) ⊏
-    r"Progress:   0%|.                    |  ETA: N/A"
+    r"Progress:   0%|. +|  ETA: .*"
     @test genmsg("", progress=1.0, width=60) == ""
     @test genmsg("", progress="done", width=60) == ""
     @test genmsgs([("", (progress = 0.1,)), ("", (progress = 1.0,))], width = 60)[end] ⊏
-    r"Progress: 100%|█████████████████████| Time: 0:00:[0-9][0-9]"
+    r"Progress: 100%|█+| Time: .*"
     @test genmsgs([("", (progress = 0.1,)), ("", (progress = "done",))], width = 60)[end] ⊏
-    r"Progress: 100%|█████████████████████| Time: 0:00:[0-9][0-9]"
+    r"Progress: 100%|█+| Time: .*"
+
+    msgs = genmsgs([
+        ("Outer", (progress = 0.0, _id = 1111)), # 1
+        ("Inner", (progress = 0.5, _id = 2222)),
+        ("Inner", (progress = 1.0, _id = 2222)), # 3
+        ("", (progress = "done", _id = 2222)),   # 4
+        ("Outer", (progress = 0.2, _id = 1111)), # 5
+        ("Inner", (progress = 0.5, _id = 2222)),
+        ("Inner", (progress = 1.0, _id = 2222)), # 7
+        ("", (progress = "done", _id = 2222)),   # 8
+        ("Outer", (progress = 0.4, _id = 1111)),
+        ("", (progress = "done", _id = 1111)),
+    ]; width=60)
```

Comment:
Just wondering whether you cover the case of inner progress with parent id's somewhere here?

Reply:
Ooooh, thanks a lot for noticing this. I totally forgot how it works :rofl:.  I added proper tests for nested progress bars now.

Reply:
Looks good!